src/geometry/cached_system.hpp

```diff
@@ -304,7 +387,7 @@ class Cached {
     const int offset = !time_dependent_;
     SPACETIMELOOP2(mu, nu) {
       const int flat =
-          nvar_deriv_ * Utils::Flatten2(mu, nu, NDFULL) + idx_[loc].dg - offset;
+          nvar_deriv_ * Utils::Flatten2(mu, nu, NDFULL) + idx_[cent_].dg - offset;
```

Comment:
What is the origin of this change? It looks like `loc` is now not being used. Should this always return the cell centered value for the metric derivative?

Reply:
This is a mistake. I will change it back to `loc`.

Reply:
Resolved.